cmd/branchingconfigmanagers/release-jobs-config-manager/internal/bumper/default-bumper.go

```diff
@@ -0,0 +1,91 @@
+package bumper
```

Comment:
You can remove this as well.

The tool should construct a ci-operator config as an object, make the changes in the struct and then bump using https://github.com/openshift/ci-tools/blob/master/pkg/config/load.go#L242

Reply:
Let's consider [this](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml) config to bump from version 4.11 to 4.12; if I have understood it correctly, I should have read this file using something like [this](https://github.com/openshift/ci-tools/blob/master/pkg/config/load.go#L271) (or similar) and then making the changes on the object. Since the version "4.11" is placed into a lot of different fields, how can I know all of them beforehand? What if people start putting version into a field we have not taken into the account? At that point I could use the reflection to overcome that problem, but what would then be the benefits of doing so instead of my actual approach? Furthermore, the function you are mentioning seems to 'only' write the file with doing the actual version bumping, am I missing something?

Reply:
Yes. You can load the config as a struct, change the required versions and then write the config to a file. The fields that hold the version should be specific and we can make a list of them.

Reply:
After a brief discussion with @droslean I agreed to follow his approach. The main motivation behind the decision is that config file could be represented in other formats than just plain text files. So the assumption to read them line by line could not work in every scenario.